nltk/test/unit/translate/test_bleu.py

```diff
@@ -138,7 +140,9 @@ def test_case_where_n_is_bigger_than_hypothesis_length(self):
         hypothesis = 'John loves Mary'.split()
         n = len(hypothesis) + 1 #
         weights = [1.0/n] * n # Uniform weights.
-        self.assertAlmostEqual(sentence_bleu(references, hypothesis, weights), 0.7165, places=4)
+        # Since no n-grams matches were found the result should be zero
+        # exp(w_1 * 1 * w_2 * 1 * w_3 * 1 * w_4 * -inf) = 0
+        self.assertAlmostEqual(sentence_bleu(references, hypothesis, weights), 0.0, places=4)
```

Comment:
This example only has a punctuation difference, it should not be evaluated as 0 BLEU too =(